src/ui/components/DefinitionList/index.js

```diff
@@ -0,0 +1,50 @@
+/* @flow */
+import * as React from 'react';
+import makeClassName from 'classnames';
+
+import './styles.scss';
+
+
+type DefinitionItemProps = {|
+  children: React.Node,
+  className?: string,
+  title: React.Node,
+|};
+
+export const DefinitionItem = (
+  { children, className = '', title }: DefinitionItemProps = {}
+) => {
+  return (
+    <React.Fragment>
```

Comment:
Happy to use `<>` here, but I wanted to push this for review last night and did't check to make sure our Babel supported it. I'll do that and update if it's easy, or we can just address it in #4345.